.prettierrc.js

```diff
@@ -2,6 +2,7 @@ const prettier = require("kcd-scripts/prettier");
 
 module.exports = {
   ...prettier,
+  arrowParens: "always",
```

Comment:
Do we need to run the files through Prettier to follow to new rule? Or do we have no occurrence of this yet?

Reply:
I've run the files through Prettier, but they're already compliant. 🙂
We don't have functions with only 1 argument that has no default value (yet) 😉